Fix instructions for libp2p in browser example

Name: libp2p-in-the-browser
Version: 0.1.0
Node Version: v11.1.0
NPM Version: 6.4.1
Platform: Linux IMCHLT047 4.15.0-43-generic #46-Ubuntu SMP Thu Dec 6 14:45:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

#### Severity: Medium

#### Description: when followed the steps from https://github.com/libp2p/js-libp2p/tree/master/examples/libp2p-in-the-browser i get the below error.

#### Error: $ npm start

> libp2p-in-the-browser@0.1.0 start /home/seshachalamm/js-libp2p/examples/libp2p-in-the-browser/1
> npm run bundle && npm run serve


> libp2p-in-the-browser@0.1.0 bundle /home/seshachalamm/js-libp2p/examples/libp2p-in-the-browser/1
> browserify src/index.js > public/bundle.js

Error: Cannot find module 'fsm-event' from '/home/seshachalamm/js-libp2p/src'

#### Steps to reproduce the error:
Follow the steps from https://github.com/libp2p/js-libp2p/tree/master/examples/libp2p-in-the-browser

--- a/examples/libp2p-in-the-browser/README.md
+++ b/examples/libp2p-in-the-browser/README.md
@@ -4,7 +4,15 @@ One of the primary goals with libp2p P2P was to get it fully working in the brow
 
 # 1. Setting up a simple app that lists connections to other nodes
 
-Simple go into the folder [1](./1) and execute the following
+Start by installing libp2p's dependencies.
+
+```bash
+> cd ../../
+> npm install
+> cd examples/libp2p-in-the-browser
+```
+
+Then simply go into the folder [1](./1) and execute the following
 
 ```bash
 > cd 1

